misc/MajorityVoting.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
<p>Add comment to explain numpy-fu. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Modified code <a href="https://sebastianraschka.com/Articles/2014_ensemble_classifier.html" target="_blank">from this blog </a>for Majority Voting. </p><p> </p><ul><li><code>np.bincount(classes_[:,0])</code> -&gt; returns bin-count for each prediction class for a given output. So if out of five classifiers one said <code>positive[0]</code> and one said <code>neutral[1]</code> and three said <code>negative[2]</code> the output would look like <code>array([1, 1, 3])</code></li><li><code>npargmax</code> -&gt; get the pred which has the max bin-count.</li></ul><p> </p><p>^ Will also add the following in the code</p>

Reply:
<p>Added in 4d31698</p>